base/CMakeLists.txt

```diff
@@ -551,6 +553,7 @@ SET(UT_FILES
 	test/QRReader_tests.cpp
 	test/textoverlayxform_tests.cpp
 	test/pipeline_tests.cpp
+	test/valvemodule_tests.cpp
```

Comment:
The spelling for    test/valvemodule_tests.cpp is wrong "M" should be caps, it is failing the compile phase